experimental/LieAlgebras/src/LieAlgebraHom.jl

```diff
@@ -71,7 +71,7 @@ end
 
 function Base.show(io::IO, ::MIME"text/plain", h::LieAlgebraHom)
   io = pretty(io)
-  println(io, LowercaseOff(), "Lie algebra morphism")
+  println(IOContext(io, :supercompact => true), h)
```

Comment:
This ensures that this detailed printing method is identical to that for `LieAlgebraModuleHom` or `GAPGroupHomomorphism`